admin/prerelease.md

```diff
@@ -0,0 +1,15 @@
+# Prereleasing New Version of Docusaurus
+
+[![asciicast](https://asciinema.org/a/n8rm53yfNURlIreGJkdSTgfAi.png)](https://asciinema.org/a/n8rm53yfNURlIreGJkdSTgfAi)
+
+# Steps
+
+1. Ensure that `origin` remote is your Docusaurus fork. Example
```

Comment:
Before executing any of these steps, we will have to make the changes in the CHANGELOG, etc, is that right? If so, we should add it before this step.

Reply:
Yes we had to make changes in CHANGELOG, etc as well.

However, for CHANGELOG I propose that we now start adding changelog everytime we merge PR just like how `jest` do it. This will make it easier for our process.

Example:
A recently closed PR in jest https://github.com/facebook/jest/pull/6776/files has a changelog commit as well
![screenshot_20180731-070432](https://user-images.githubusercontent.com/17883920/43429916-74a244ea-9490-11e8-8fc5-0dc532164922.png)

Reply:
I'm not opposed to trying that but feel that it's hard for Docusaurus PR submitters to write the CHANGELOG. We'll need to create a template for the next version and pending PRs would always have to take note of which section of CHANGELOG they're updating. Might be a source of frequent merge conflicts. @JoelMarcey thoughts?

Reply:
Cc @JoelMarcey on changelog